zebra/interface.c

```diff
@@ -2848,12 +2840,10 @@ DEFUN (bandwidth_if,
 	return CMD_SUCCESS;
 }
 
-DEFUN (no_bandwidth_if,
-       no_bandwidth_if_cmd,
-       "no bandwidth [(1-100000)]",
-       NO_STR
-       "Set bandwidth informational parameter\n"
-       "Bandwidth in megabits\n")
+DEFUN(no_bandwidth_if, no_bandwidth_if_cmd, "no bandwidth [(1-1000000)]",
```

Comment:
again please do not change the whitespace here.  clang-format is just wrong here

Reply:
you mean, lets go back to previous format of DEFUN ?

Reply:
Yes

Reply:
ok